libs/@guardian/source-react-components-development-kitchen/src/numeric-input/NumericInput.tsx

```diff
@@ -134,7 +134,7 @@ export const NumericInput = ({
 					id={textInputId}
 					aria-required={!optional}
 					aria-invalid={!!error}
-					aria-describedby={error || success ? descriptionId(textInputId) : ''}
+					aria-describedby={error ?? success ? descriptionId(textInputId) : ''}
```

Comment:
Is this really the behaviour we want? I can see the fallback when `success` is falsey to be an empty string rather than `undefined`.